src/theme.ts

```diff
@@ -36,23 +38,76 @@ const palette = {
   orange: "#D4450C"
 } as const;
 
+const buttonVariants = {
```

Comment:
The button config can go in a button specific file, otherwise you'll just be putting everything here

Reply:
It doesn't feel to me like it's hitting that simplicity tradeoff threshold yet (could see theme just composing typography, palette, etc. later if those expand) but I don't feel very strongly about it, I can change it.